### DISCUSSION TITLE
テーブルのcaptionを追加できるようにしてほしい

### DISCUSSION THREAD

**Discussion by NagayamaToshiaki** [increments/qiita-discussions#274] [要望・アイデア・不具合]
テーブル説明の `<caption>` 要素を利用したいです。markdownでは指定できず、直接htmlコードを記述して `<caption>` を挿入した場合、通常の段落として処理されてしまいます。パーサーの都合というのもあると思いますが、テーブル説明で正式な見出しが使えないのは残念です。ご対応お願いいたします。

**Comment by tanaka-asami415** [comment]
@NagayamaToshiaki 
ご意見ありがとうございます🙏
一度調査してから、ご返答させていただきます🙇
お休みを挟むため、少しお時間がかかってしまうかもしれませんが、お待ちいただけますと幸いです🙏💦

**Reply by tanaka-asami415** [reply]
@NagayamaToshiaki 
お待たせいたしました🙇
直接htmlコードを記述した場合は、`<caption> `が反映されるよう対応いたします💪
反映まで、お待ちいただけますと幸いです🙇

**Comment by wataru86** [comment]
@NagayamaToshiaki
お待たせいたしました :bow:
HTMLでテーブルを入力する際に `<caption>` 要素が使えるようになりました！

詳細は [リリースノート](https://qiita.com/release-notes#2022-09-20) や [Markdown記法 チートシート](https://qiita.com/Qiita/items/c686397e4a0f4f11683d#%E3%83%86%E3%83%BC%E3%83%96%E3%83%AB) をご参照ください。
